src/components/Product/Images/index.tsx

```diff
@@ -0,0 +1,17 @@
+import { ImagemProduto } from "../../../data"
+import styles from './Images.module.scss'
+
+const Images = ({ images }: { images: ImagemProduto[] }) =>
```

Comment:
Aqui o nome do component ser Images ficou genérico demais, vai parecer que posso usar o componente para qualquer imagem, quando na verdade, você espera que sejam as imagens dos produtos, então varia até mesmo ter um nome mais específico, se for o caso das imagens animadas, poderia ser ProductImagesSlider ou ProductImagesCarousel